api/src/services/organizations_v1/update_user_organization_roles.py

```diff
@@ -0,0 +1,91 @@
+import logging
+from typing import Sequence
+from uuid import UUID
+
+from sqlalchemy import select
+
+from src.adapters import db
+from src.api.route_utils import raise_flask_error
+from src.auth.endpoint_access_util import can_access
+from src.constants.lookup_constants import Privilege, RoleType
+from src.db.models.entity_models import Organization
+from src.db.models.user_models import (
+    LinkRoleRoleType,
+    OrganizationUser,
+    OrganizationUserRole,
+    Role,
+    User,
+)
+from src.services.organizations_v1.get_organization import get_organization
+
+logger = logging.getLogger(__name__)
+
+
+def get_role(db_session: db.Session, role_ids: set[str]) -> Sequence[Role]:
```

Comment:
Nit: To go along with the request schema change / pluralize function for readability
```suggestion
def get_roles(db_session: db.Session, role_ids: set[uuid.UUID]) -> Sequence[Role]:
```